system/ee/language/english/fieldtypes_lang.php

```diff
@@ -75,19 +75,19 @@
 
     'rel_ft_display_entry_id_desc' => 'When enabled, entry IDs will be displayed together with entry title inside the field.',
 
-    'rel_ft_max' => 'Maximum selection',
+    'rel_ft_max' => 'Maximum relationship',
```

Comment:
possibly change to "Maximum number of related entries"

Reply:
Gut reaction is that I like this.